modules/leaderboard_ui.js

```diff
@@ -0,0 +1,56 @@
+import LeaderBoardService from './leaderboard_service';
+import User from './user';
+
+const list = document.querySelector('.scores-list');
+const refreshButton = document.querySelector('.refresh-button');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
+const submitButton = document.querySelector('.submit');
+
+const createElement = (data, isWhite) => {
+  const element = document.createElement('p');
+  element.classList.add('item');
+  if (isWhite) {
+    element.classList.add('white');
+  }
+  element.innerHTML = `${data.user} : ${data.score}`;
+  return element;
+};
+const renderItems = async () => {
+  const result = await LeaderBoardService.getData();
+  /** @type {Array} */
+  const data = result.result;
+
+  for (let i = 0; i < data.length; i += 1) {
+    const element = createElement(data[i], i % 2 === 0);
+    setTimeout(() => {
+      list.appendChild(element);
+    }, 400 * i);
+  }
+};
+
+refreshButton.addEventListener('click', () => {
```

Comment:
- [ ] When I click the refresh button after submitting my entries the score list doesn't alter. No user data is reflected on the scoreboard.  Please rectify this issue.

Reply:
@FrederickMih  okay i will send u after 30 min, but anyways, i deployed the page and it works in the deployed page.. check again please:
https://yosefhesham.github.io/leaderboard/dist/

Reply:
Hi @yosefHesham , you can request a review once more.

Reply:
@FrederickMih thanks for wasting my time..